distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc

```diff
@@ -0,0 +1,19 @@
+[id="distr-tracing-deploying-otel"]
+= Configuring and deploying distributed tracing data collection
+include::modules/distr-tracing-document-attributes.adoc[]
+:context: deploying-data-collection
```

Comment:
Not critical, but we usually try to match the context with the ID above. Could this also be "distr-tracing-deploying-otel"?

Reply:
Could this also be "distr-tracing-deploying-otel"?

???  That is the current ID, was that what you meant to type?

Reply:
I think that is what I typed? I was suggesting to make this change on line 4:

```
:context: distr-tracing-deploying-otel
```

Reply:
Ah, now I see what you're saying.  Since the context shows up in the URL, I wanted it to be the product name (OTEL is the upstream name).